### ISSUE TITLE
Add Domain Map for use cases

### ISSUE THREAD

**Issue by jandrieu** [w3c/did-use-cases#87]
Something like the one in the VC doc: https://w3c.github.io/vc-use-cases/#user-needs

**Comment by philarcher** [comment]
I'm leaving this to you @jandrieu ;-)

**Comment by jandrieu** [comment]
This is still on me. I've started this, including conversations with @philarcher. It still needs more work.

**Comment by philarcher** [comment]
Done. See https://w3c.github.io/did-use-cases/#domainMap

**Comment by TallTed** [comment]
tl:dr; There ain't no easy fix for the tiny label text.  :-/ 

SVG doesn't set a `font-size` for the `text` class which covers the box labels; this is inherited from the Respec stylesheet (and it's OK when the SVG is "raw").  But then the SVG gets scaled down roughly 61% from "original" size, and so does the effective `font-size`.

Changing the SVG source to use `24px` for the `text` class therein would, I believe, make the labels legible for all ... but then many of the labels overflow the boxes.

Wider boxes would make the Respec "scaledown" more severe, and require a larger `font-size`, which would force even wider boxes... wash, rinse, repeat.

Taller boxes would allow for line-breaks in the labels, but SVG 1.1 can't wrap text(!), even with manually inserted `<br />` elements.  (This might change in SVG 2, but that *appears* to have stalled; though some pages indicate they're (re?)chartered into this year, the latest editor's draft I could locate is dated Oct 2018.)

That's all I've got...

**Comment by iherman** [comment]
Well, it seems that I will have to put on my glasses when looking at the diagram :-(

**Comment by philarcher** [comment]
Needs final update but this is basically done. If I can improve the legibility, I will.
